lms/djangoapps/course_api/blocks/transformers/milestones.py

```diff
@@ -35,22 +45,39 @@ def collect(cls, block_structure):
         block_structure.request_xblock_fields('is_proctored_enabled')
```

Comment:
@nasthagiri: @dianakhuang and I agree that it would seem cleaner to have special exams (which aren't milestones), to be in its own transformer.  @dianakhuang is under the impression that you don't think that is a good idea.  Could you clarify why?  Thank you!